portfolio/src/main/webapp/googleMap.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Hello Google Maps</title>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAmZ__3ulQ2Ckxp1Me3hR1VeHjGyiiBH4Q"></script>
```

Comment:
You'll need to disable and change the key here, having it on Github now makes it public. One option could be to store keys in another file that is not checked into git. Could talk to Emma about it, same thing happened to her :)

Reply:
removed the key in this commit ac9c849a9cad028f5504b619c8a2ce8e3d0d3286